js/customscripts.js

```diff
@@ -30,6 +30,18 @@ function renderTOC() {
 
 var $versionSwitcher, versionSwitcherBottom = Infinity;
 
+const cockroachDomains = [
```

Comment:
@nickvigilante LGTM. You could also add `cockroachlabs.atlassian.net` in case we ever link to the public wiki https://cockroachlabs.atlassian.net/wiki/spaces/CRDB/overview

Reply:
I'll send a follow-up PR.